scripts/mysql/reader/create_replication.sql

```diff
@@ -0,0 +1,10 @@
+STOP SLAVE;
+CHANGE MASTER TO
+MASTER_HOST='mysql-1',
+MASTER_AUTO_POSITION=1,
+MASTER_USER='replication',
+MASTER_PASSWORD='password';
+
+start slave;
```

Comment:
Good catch! After investigation, it was found that if a race condition were to happen (replication starts before user are created on writer), the slave would retry connection. I added a custom retry time so that it doesn't block the rest of the tests (retries every second for 5 minutes before aborting).